trie/database.go

```diff
@@ -298,7 +300,11 @@ func NewDatabase(diskdb ethdb.KeyValueStore) *Database {
 func NewDatabaseWithConfig(diskdb ethdb.KeyValueStore, config *Config) *Database {
 	var cleans *fastcache.Cache
 	if config != nil && config.Cache > 0 {
-		cleans = fastcache.New(config.Cache * 1024 * 1024)
+		if config.Journal == "" {
+			cleans = fastcache.New(config.Cache * 1024 * 1024)
+		} else {
+			cleans = fastcache.LoadFromFileOrNew(config.Journal, config.Cache*1024*1024)
```

Comment:
`config.Cache` * 1024 * 1024 => `config.Cache`  * `units.MiB?` ignore if we want to minimize the diff.

Reply:
I'd prefer to minimize the diff rather than import an avalanchego constant here.